BetterStacking/Implementation.cs

```diff
@@ -72,13 +57,60 @@ internal static bool CanBeMerged(GearItem target, GearItem item)
     internal static void Log(string message) => MelonLogger.Msg(message);
     internal static void Log(string message, params object[] parameters) => MelonLogger.Msg(message, parameters);
```

Comment:
Delete these. They access obsolete methods. The logger instance is a MelonMod property.